README.md

```diff
@@ -36,24 +36,16 @@ is not published on Maven Central Repository.
-|----------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [Ontology 1.1 / 2021-June](https://github.com/IATA-Cargo/ONE-Record/tree/master/June-2021-standard-COTB-endorsed)    | not available (Note 1)                                                                | Yes (Release 1.1.x)                                                                                                                                                                |
-| [Ontology 2.0 / 2022-May](https://github.com/IATA-Cargo/ONE-Record/tree/master/May-2022-standard-forCOTBendorsement) | not available (Note 1)                                                                | Yes (Release 2.0.x)                                                                                                                                                                |
-| [Ontology 2.1 / 2022-December](https://github.com/IATA-Cargo/ONE-Record/tree/master/December-2022-standard)          | Yes                                                                                   | not available                                                                                                                                                                      |
-| Scope                                                                                                                | DataModel Ontology only                                                               | API + DataModel  Ontology                                                                                                                                                          |
-| 3rd party Java annotations                                                                                           | none (clean API)                                                                      | [Jackson](https://github.com/FasterXML/jackson), [Swagger IO](https://github.com/swagger-api/swagger.io),  [Spring Framework](https://github.com/spring-projects/spring-framework) |
-
-Note 1: Not released but can be build manually by replacing Ontology file and building manually within project.
+Since we aim to keep the version numbers in relation to the used ontology version we decided to use a slightly different pattern for the versioning.
+The most recent version of the DataModel Ontology is: **3.0.0** (as of 07-05-2024). The next release of this project will therefore get the version: **0.3.0000**. 
```

Comment:
Ich hätte gesagt wir nehmen 3.0.0000, entsprechendes dann auch in den beiden Absätze dadrunter